deploy/hostpath/README.md

```diff
@@ -0,0 +1,6 @@
+The .yaml files in this directory depend on the RBAC files that were
```

Comment:
I think the rbac rules should be copied into examples/hostpath so that a hostpath deployment can be done using the YAML files.  The docs will eventually point to the hostpath repos.

Reply:
No, please not. Having to keep files in sync in different repos is exactly the kind of maintenance headache that we are currently trying to get rid of.
 
The Example.md is self-contained without copies because it references upstream files via URL. I have a PR pending for Kubernetes E2E which achieves the same (https://github.com/kubernetes/kubernetes/pull/71703). Let's not go in the other direction here.

The deployment here won't be complete, but it doesn't have to be. That's what the Example.md is for - IMHO.

Reply:
FWIW, I personally would have preferred a solution where all relevant .yaml files are in the same repo because then they can be updated and tested together. But it was decided that each component maintainer is also responsible for maintaining the corresponding .yaml files (in particular the RBAC files), which is why we now move the individual files away from centralized places (docs, kubernetes E2E) into individual repos.

Reply:
Agree with your last comment re: having all yamls in the same repo.  My thought was to have all yamls that are necessary to standup a hostpath instance under `hostpath/example` instead of `docs/examples`  to minimize.  I think when the config yamls are located in docs/examples, it may not be clear that they need updated as hostpath changes.

Reply:
Moving from kubernetes-csi/docs is good. Then eventual kubernetes-csi/docs will just reference files elsewhere by URL.

But do you really mean *all* yamls, including the RBAC ones from driver-registrar, external-attacher and external-provisioner? That goes against the decision made earlier, so before you do that, discuss it in the CSI standup.

In the meantime, can we merge this PR?

Reply:
We should leave RBAC files in sidecar container repos (external-provisioner, external-attacher, etc.).
Each driver can provide it's own YAMLs for DeamonSet/ReplicaSet, along with a deployment script that deploys all the YAMLs (including the RBAC ones).